client/my-sites/theme/controller.jsx

```diff
@@ -85,15 +85,16 @@ export function details( context, next ) {
 		themeSlug: slug,
 		contentSection: context.params.section,
 		title: decodeEntities( title ) + ' — WordPress.com', // TODO: Use lib/screen-title's buildTitle. Cf. https://github.com/Automattic/wp-calypso/issues/3796
-		isLoggedIn: !! user
+		isLoggedIn: !! user,
+		site: context.params.site_id,
```

Comment:
Can we handle this by `connect()`ing the `ThemeDetailsData` component to the `getSelectedSite()` selector? I think we should try to move away from manually parsing `context.params` for `site_id`, since there's a whole middeware dedicated to that (which also does some validation etc, and updates Redux state accordingly).